src/glwidget.cpp

```diff
@@ -2144,11 +2044,7 @@ void GLWidget::resizeGL(int w, int h)
     glViewport(0, 0, (GLint)w, (GLint)h);
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
-    if (!ortho) {
-        gluPerspective(60 * mScale, (GLfloat)w / (GLfloat)h, 0.0001, 10000.0);
-    } else {
-        gluOrtho2D(0.0, (GLdouble)w, 0.0, (GLdouble)h);
-    }
+    gluPerspective(60 * mScale, (GLfloat)w / (GLfloat)h, 0.0001, 10000.0);
```

Comment:
Why's this removed?

Reply:
The global variable that switched to the other branch of the `if` is never set - so the other part is unreachable.